hera_qm/utils.py

```diff
@@ -245,14 +245,18 @@ def get_metrics_ArgumentParser(method_name):
                         'to full flag array and apply before delay filter.')
     elif method_name == 'xrfi_run':
         ap.prog = 'xrfi_run.py'
-        ap.add_argument('--ocalfits_file', default=None, type=str, help='Omnical '
-                        'calfits file to use to flag on gains and chisquared values.')
-        ap.add_argument('--acalfits_file', default=None, type=str, help='Abscal '
-                        'calfits file to use to flag on gains and chisquared values.')
-        ap.add_argument('--model_file', default=None, type=str, help='Model visibility '
-                        'file to flag on.')
+        ap.add_argument('--ocalfits_files', default=None, type=str, help='Omnical '
+                        'calfits files to use to flag on gains and chisquared values.',
+                        nargs='+')
+        ap.add_argument('--acalfits_files', default=None, type=str, help='Abscal '
+                        'calfits files to use to flag on gains and chisquared values.',
+                        nargs='+')
+        ap.add_argument('--model_files', default=None, type=str, help='Model visibility '
+                        'file to flag on.',
+                        nargs='+')
         ap.add_argument('--data_file', default=None, type=str, help='Raw visibility '
-                        'data file to flag on.')
+                        'data files to flag on.',
```

Comment:
If the other variables are being pluralized, this one should be too.

Reply:
yep.